sbws/util/config.py

```diff
@@ -21,6 +25,12 @@
 log = logging.getLogger(__name__)
 
 
+def _create_user_config_file(fname):
+    """Copy minimal user config to user config path."""
+    if not os.path.isfile(fname):
+        shutil.copyfile(MINIMUM_USER_CONFIG_PATH, fname)
```

Comment:
Do you think we should assert on `fname` not existing here? I think so. 

If the file exists already, we don't create/overwrite anything. We silently do nothing. As a programmer, I'd expect a function with "create" in its name to do *something* when the file already exists. I would expect it to do (at least) one of the following.

- overwrite it
- warn about it existing (and about overwriting it, if it does so)
- assert that the file doesn't exist

I would prefer the assert. If not that, then a different function name. If not that, a debug log warning that it exists.